docs/hello-world-docs/hello-pt/doc.py

```diff
@@ -0,0 +1,358 @@
+
```

Comment:
now I read this doc.py again, 
Since we can't simply execute this file using python doc.py
I suggest we use ".rst" file directly but include the codes using literalinclude, so something like:



```
Model
-----

In PyTorch, models are defined as subclasses of :class:`torch.nn.Module`. The model implementation is located in ``model.py``.

.. literalinclude:: model.py
    :language: python
    :linenos:
    :caption: model.py


```